src/index.js

```diff
@@ -1,86 +1,110 @@
-const db = require('./notifications/db');
-const Configuration = require('./notifications/config');
-const notificationController = require('./notifications/controller');
+import db from './notifications/db';
+import Configuration from './notifications/config';
+import notificationController from './notifications/controller';
 
-function Notification(config) {
-  this.config = new Configuration(config);
-  this.configDatabase = this.config.database;
-  this.baseApi = this.config.baseApi;
-}
+class Notification {
+  constructor(config) {
+    this.config = new Configuration(config);
+    this.configDatabase = this.config.database;
```

Comment:
What are the advantages of use this.configDatabase instead this.config.database internally